src/components/DatePickerInline/YearPicker.tsx

```diff
@@ -1,6 +1,6 @@
 import { useRef, useEffect, memo, useMemo } from 'react';
 import { StyleSheet, View } from 'react-native';
-import type { FlashList } from '@shopify/flash-list';
+import type { FlatList as GestureHandlerFlatList } from 'react-native-gesture-handler';
 import { useComponentStyles, useMolecules } from '../../hooks';
```

Comment:
DatePickerInline has a file Swiper.native.tsx which renders a ScrollView that comes from react-native

Surprisingly, the View also in this file also comes from react-native

Reply:
:)
I must've overlooked.

Reply:
Updated